FileAPI/reading-data-section/filereader_result.html

```diff
@@ -54,6 +55,61 @@
 
       readArrayBuffer.readAsArrayBuffer(blob);
     }, "readAsArrayBuffer");
+
+    async_test(function() {
+      var readBinaryString = new FileReader();
+      assert_equals(readBinaryString.result, null);
+
+      readBinaryString.onloadend = this.step_func(function(evt) {
+        assert_equals(typeof readBinaryString.result, "string", "The result type is string");
+        assert_equals(readBinaryString.result, "This test the result attribute", "The result is correct");
+        console.log(readBinaryString.result);
+        this.done();
+      });
+
+      readBinaryString.readAsBinaryString(blob);
+    }, "readAsBinaryString");
+
+
+    function resultDuringLoadTest(readMethod, event) {
+      async_test(function() {
+        var reader = new FileReader();
+        assert_equals(reader.result, null);
+
+        reader.addEventListener(event, this.step_func_done(function(evt) {
+          assert_equals(reader.result, null);
+        }));
+
+        reader[readMethod](blob);
+      }, 'result is null during "' + event + '" event for ' + readMethod);
+
+      async_test(function() {
+        var reader = new FileReader();
+        reader.onloadend = this.step_func(function(evt) {
```

Comment:
Should this null out `reader.onloadend` so it isn't trying to run this code a second time (after done, in theory...)

Reply:
Changed to promise_test using EventWatcher, so no longer applicable.